### PR TITLE
TEST: ucp_client_server.c remove progress call from server's conn_req_cb

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 3 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 30-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/e48505f3fbd432065af73c676f0f6516f1214d52/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/a6547988c7152ee79a49ef61f6f7e62f85b52f25/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/4a1982bc0766a793060a424c3d9be3577a74532b/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/ef65c599868886704bbb4b1ddd0042eeb9c8d37a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/f7a9ce33201a689f5015a231d3e02d68055adca6/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/9c0492161bf0d3a5a4027b00c8de2c5292e13c24/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/88e28ab63012f3044722a09a6ffe579fd2d11497/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/1afdc144cb5695ed6876a5c28cc626628cfd2ac3/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/b0b969aa98861c3573119f3fd90a068cddf8ba6e/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/9e5e32851e12df2eae63e048a4b380484077bfcd/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/6d075d550564bb06f7fa4a31b0e7064303a648f8/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/78276407c68417aff657e824b7e8d4cacfe0a166/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/beff3e50583ae65a20ec1d706f17b0397eb71ff0/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/d62d0373fb1166c0df2146243f5ee565fff2b437/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/0cb024842d037e99f4d31b133e5e1aecc71023a4/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/8b7307a081d61812dbb65ae3537eb6225e0b37d5/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/5e023cda189530ba0a2875c50e7aee9e136ff028/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/017cf0e6ff1eb3d2b72190846b97e8d297306e7d/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/04f572f2a6bf974f6be2eb8ec57beb0770b1f85a/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/6579c42e5020f4650119b2a134f5a1541ac0e465/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/2867af11a403d2ee97e9dfee798c2c4ca8fb38e4/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/b7017ab535f6e91822ea53474d071590da307381/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/d631b0ffda75f3824f933df6b6ad4c0addbc41ee/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/e3a96fc95c36fe9bb3025e6b3e5a3d51fbacf2f1/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/bba25d334839d9877be10c589e99504e/raw/47a592eff613288503b944fca7a8a7695622e254/hpc-test-node-new_W0) | :question: UNKNOWN

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 03-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/66912628e0bc0a9c6bf5914b6b6182e18976f48b/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/034b2e437887557e5952ad4332edd9050a642cdd/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/6a5daabd22a33628fa7923ed6e592dba49913db3/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/94afd73b0c4239a14249ac0ff4cb874ec24f0b04/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/6c501839a3fb9b291b332c7adad4b0609eb0ed6f/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/23a66d6560f6694f20b073d2c184c7c1c8d8a5ae/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/1cf441f1c1c9b5de2c721f32f0fb2d099a237d2c/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/42f69299f427d100bf8dda2c2bbfeaac43d17e07/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/96d02859bdbded76974d59deb8fe42ec16c15d79/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/e8e9bfe8013c128098dc6f8a9b3df347e293ec22/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/77a0efd59e6b4dcd65277753675bec2b9f86fb39/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/3e9c823009b352af925821479ebc3c9dbfd19009/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/b548128f9ae654ec254e5d28eebdc639e00ecded/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/0ad69107d0bd8fd71e2b788f08059f72ebc4eab7/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/d583c17f28d0ef9ba7b532fe4567778d8e031ea4/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/d844b9975cb878fec7dc79bcf6e144829c5de839/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/d035cfa2f851893c568715f0f05aeec73cf011f9/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/c9c2a988c308c63df359033a807176198162e68c/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/7805144fb30622e9c57529c64e554d6a038adf74/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/5a284da26396813f5f76ffe674696a220c3de832/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/d64c393c1662a3cc420fa3eeae3cca778724d8f6/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/aea907390d75f00f045a1fb70c522d825f57694d/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/2a79939bd6577b55f153499370c2e8c004c1b04a/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/66365e4fd03febc2b0e79417d17a12ea9cdab15d/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1226010107855a42faf106c066834f20/raw/19ec3f22e6e4e06ae0c74fd1ace3b32bd7c297b9/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: test/examples/ucp_client_server.c [hunk 1]
```diff
@@ -584,23 +597,39 @@ int main(int argc, char **argv)
         }
 
         /* Server is always up listening */
-        printf("Waiting for connection...\n");
         while (1) {
-            ucp_worker_progress(ucp_worker);
+            /* Wait for the server to receive a connection request from the client */
```
Comment (yosefe) [inline test/examples/ucp_client_server.c]: pls extract all server side code to new function run_server()

File: test/examples/ucp_client_server.c [hunk 2]
```diff
@@ -411,12 +425,16 @@ static int init_worker(ucp_context_h ucp_context, ucp_worker_h *ucp_worker)
 static void server_conn_handle_cb(ucp_conn_request_h conn_request, void *arg)
 {
     ucx_server_ctx_t *context = arg;
-    ucp_ep_h         ep;
-    ucp_ep_params_t  ep_params;
-    ucs_status_t     status;
-    ucp_worker_h     data_worker;
 
-    data_worker = context->data_worker;
+    context->conn_request = conn_request;
```
Comment (yosefe) [inline test/examples/ucp_client_server.c]: as we discussed, pls check context->conn_request == NULL

File: test/examples/ucp_client_server.c [hunk 3]
```diff
@@ -584,23 +597,39 @@ int main(int argc, char **argv)
...
+            }
+
+            /* Server creates an ep to the client on the data worker.
+             * This is not the worker the listener was created on.
+             * The client side should have initiated the connection, leading
+             * to this ep's creation */
+            status = server_create_ep(ucp_data_worker, context.conn_request,
+                                      &server_ep);
+            if (status != UCS_OK) {
+                fprintf(stderr, "failed to create a server endpoint\n");
```
Comment (yosefe) [inline test/examples/ucp_client_server.c]: print status string

File: test/examples/ucp_client_server.c (hunk 2 above)
Comment (evgeny-leksikov) [reply test/examples/ucp_client_server.c]: how it supposed to be handled? if reject then it should be handled on client

File: test/examples/ucp_client_server.c [hunk 4]
```diff
@@ -425,8 +426,21 @@ static int init_worker(ucp_context_h ucp_context, ucp_worker_h *ucp_worker)
 static void server_conn_handle_cb(ucp_conn_request_h conn_request, void *arg)
 {
     ucx_server_ctx_t *context = arg;
+    ucs_status_t status;
 
-    context->conn_request = conn_request;
+    if (context->conn_request == NULL) {
+        context->conn_request = conn_request;
+    } else {
+        /* The server is already handeling a connection request from a client,
```
Comment (yosefe) [inline test/examples/ucp_client_server.c]: typo

File: test/examples/ucp_client_server.c [hunk 5]
```diff
@@ -425,8 +426,21 @@ static int init_worker(ucp_context_h ucp_context, ucp_worker_h *ucp_worker)
...
+         * reject this new one */
+        printf("Rejecting a connection request. "
+               "One client at a time is supported.\n");
```
Comment (yosefe) [inline test/examples/ucp_client_server.c]: "Only one .. "

File: test/examples/ucp_client_server.c [hunk 6]
```diff
@@ -505,6 +519,95 @@ static ucs_status_t start_server(ucp_worker_h ucp_worker,
     return status;
 }
 
+static int run_server(ucp_context_h ucp_context, ucp_worker_h ucp_worker,
+                      char *listen_addr)
+{
+    ucx_server_ctx_t context;
+    ucp_worker_h     ucp_data_worker;
+    ucp_listener_h   listener;
```
Comment (yosefe) [inline test/examples/ucp_client_server.c]: can we remove "listener" variable and use only "context.listener"

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 10 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 04-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/dabbe0e4b8033ba25d67ee2195ffed765372a8b0/_main) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/47aff82630525580cb05f2eb7f00442e1eb9aa31/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/89cc21e82a7344662ae2686e0b59a5e757461370/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/38585bc8cf99a893554b2efb709cdc0019c1d600/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/6ff30779244640f22fd038413630dfbc6f30ff7d/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/e9a4b147853739478d4ca28247aae67cfa4943f0/hpc-test-node-legacy_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/ccb690c13385a1b69f625d3fca6d13546802fbc5/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/28ece86eaf24819842d54b4449bdac6b4ac66d4d/r-vmb-ppc-jenkins_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/e2da7371bed196d88e43c827f958726a0018897d/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/b20cbb9c709cc26945cd8c2045a3f00d84bb5b1b/r-vmb-ppc-jenkins_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/2daf5adcf49ea5d3edf392436cee6e922dbec66e/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/8adc13490c7a153531fa27a12a0c38d51133d727/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/1495c47b5d8debaea20ec350eadf4b0b5c033f9b/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/9b985a6ba826d06e585f2b858b36334099c9a3c9/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/0e963fe456fb9237e2b2557b2536dd27dab4211c/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/d923a4ea1ff70a03c917e1916ff0e3b33ff60957/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/198a43cc5ff3a553c5a010d5ffb313b29d9080ed/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/b571de43bdfcf65b65ed654d49ad92c618bd86e7/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/dd4a6cb8d6bb273d6b8465dfb165d5d74c99f913/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/220fddd83a4206b6494a3c01d329dad0dbbde6e5/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/638eb88e7c5ff6660f313d809fb236ee6773f231/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/0f0e370565a2919a745aad16a8b77242f0510964/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/c7ac7b608c6359dc4d5dc0ca63ea4e442b2c1e01/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/32827e5ddc3cb5d182990b1e0be71b64d12aa7ce/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/84f9ac15712ee446066b9ab9f4f74c81/raw/17691403560692691c6ea0a7c63f39011515f410/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 04-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/3edfe93d6d3a6ef56a890c5416af66a685801f23/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/2ef8f4bf2715e948433426f9e0480af2e00617df/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/0483760819e1155322f90525920945ee0664f830/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/fb290e4e2625bb2c8dbb50d2c6f197987f1f4587/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/94adb3f71a1065909fdf2c628eba5c351352b9a8/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/d3fceb4153efdce7d65ff29ece40f68e3f376091/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/862d1f17cc3173fb2f9a589d9e0dca9d25604cce/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/1c35605656de4c8969b87c0001e5448ac9f30604/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/37e3405208b5be59d4660de1f44ad5e0e09b556e/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/9bfa5c761aabffb28cc1009db9e9fd40e9a68d7d/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/c1cbcdcedca49a3be54005115b8b80bef0a15481/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/191a46dfdd2706a2096e62eabc35dfea0082bf53/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/c657fc772d0f36723aebffb1e847f55645c5f503/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/5892da9e09f9102a0a0a8e6ab8c70e45e09603e3/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/056816ad4d5098c5bf30be8a43e100d3743c88f5/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/3e68c697f5651ceaae1f7f87ac7e6239545a87ff/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/542a814ef1715e148caa16928cec761c3d72d035/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/c6454292bb21358f0dd2e700aea27f7e8c3ac8c0/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/34cfd9582622adb90d6d0ebad35381a1a3a383b5/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/ec9115eb43202b89d54d5e823ccde45c641add06/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/0e598a0a733b92a5996d1ad94ac1abf3ec7f7d42/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/d2ae57a9b11d39dff3ae0927d73919a2142ff747/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/d996e3b4bbef0d45a7b4bb95981a845a9c657a55/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/1b5c1bfa22446a16778d55f05e48822c89f402b8/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/ca085c8e4fe6fcb88127de24152e95ac/raw/8495d8b1bb1b059574aae2ebbb0262b276675017/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**